bindings/core/src/method_handler/secret_manager.rs

```diff
@@ -124,6 +124,13 @@ where
             if let Some(secret_manager) = secret_manager.downcast::<StrongholdSecretManager>() {
                 secret_manager.store_mnemonic(mnemonic).await?;
                 Response::Ok
+            } else if let Some(secret_manager) = secret_manager.downcast::<SecretManager>() {
```

Comment:
Was there still an issue with this? I think I saw a discussion somewhere

Reply:
There was only an issue without this being added